mkdocs.yml

```diff
@@ -28,9 +28,9 @@ markdown_extensions:
     - toc:
         permalink: true
     - attr_list
-    - pymdownx.emoji:
-        emoji_index: !!python/name:materialx.emoji.twemoji
-        emoji_generator: !!python/name:materialx.emoji.to_svg
+    # - pymdownx.emoji:
+    #     emoji_generator: ""
+    #     emoji_index: ""
```

Comment:
Could you share the reasons to remove this block?

Reply:
oh I thought that is what you meant [this](https://github.com/testcontainers/testcontainers-go/pull/1190#discussion_r1193614288) you didn't think it needed to be there. Was this there before? If so then uncommenting leads to the build failing with this error `Config value: 'markdown_extensions'. Error: unsupported callable`